src/Page/Options.elm

```diff
@@ -271,21 +201,54 @@ viewResultItemDetails channel item =
 
                 Just value ->
                     wrapWith value
+
+        wrapped wrapWith value =
+            case value of
+                "" ->
+                    wrapWith <| "\"" ++ value ++ "\""
+
+                _ ->
+                    wrapWith value
+
+        showDetails =
+            if Just item.source.name == show then
+                [ div []
+                    [ div [] [ text "Default value" ]
+                    , div [] [ withEmpty (wrapped asPreCode) item.source.default ]
+                    , div [] [ text "Type" ]
+                    , div [] [ withEmpty asPre item.source.type_ ]
+                    , div [] [ text "Example" ]
+                    , div [] [ withEmpty (wrapped asPreCode) item.source.example ]
+                    , div [] [ text "Declared in" ]
+                    , div [] [ withEmpty asGithubLink item.source.source ]
+                    ]
+                ]
+
+            else
+                []
+
+        open =
+            SearchMsg (Search.ShowDetails item.source.name)
     in
-    dl [ class "dl-horizontal" ]
-        [ dt [] [ text "Name" ]
-        , dd [] [ withEmpty asText (Just (encodeHtml item.source.name)) ]
-        , dt [] [ text "Description" ]
-        , dd [] [ withEmpty asText item.source.description ]
-        , dt [] [ text "Default value" ]
-        , dd [] [ withEmpty (wrapped asPreCode) item.source.default ]
-        , dt [] [ text "Type" ]
-        , dd [] [ withEmpty asPre item.source.type_ ]
-        , dt [] [ text "Example value" ]
-        , dd [] [ withEmpty (wrapped asPreCode) item.source.example ]
-        , dt [] [ text "Declared in" ]
-        , dd [] [ withEmpty asGithubLink item.source.source ]
+    [ li
+        [ class "option"
+        , onClick open
+        , Search.elementId item.source.name
         ]
+        ([]
+            |> List.append showDetails
```

Comment:
This is completely unnecessary as it is equivalent to:


```suggestion
        (showDetails
```

Following the monoid identity law of list

> There exists an element e in S such that for every element a in S, the equations e • a = a and a • e = a hold.

in this case `e = []` and `• = List.append`